update-since-todo.sh

```diff
@@ -19,7 +21,7 @@ do
     lineSha=$( git blame --porcelain -L "$line,$line" "$file" | head -1 | cut -d ' ' -f 1 )
     echo -e "\tfirst sha: $lineSha"
 
-    firstTag=$( git tag --sort=creatordate --contains "$lineSha" | head -1 )
+    firstTag=$( git tag --sort=creatordate --contains "$lineSha" 'jenkins-*' | head -1 )
```

Comment:
Should we delete the untagged-.* junk tags that have been created by Release Drafter mistakes?

Reply:
Don't know, but I'd rather not have to rely on it not being there, so this change probably should stay either way.